### DISCUSSION TITLE
레벨2 글쓰기 워크숍 회고 | 2023.05.24

### DISCUSSION THREAD

**Discussion by Wonmi91** [woowacourse/retrospective#33] [General]
워크숍 과정 전체를 돌아보며 의미있었던 지점이나 순간들에 대해 떠올리며 회고한다.

조 별로 회고한 내용을 정리해 제출해주세요 : )

**Comment by woosung1223** [comment]
### 4조 글쓰기 DAY2 회고

- 비버: 다른사람의 경험을 글로 다양하게 접할 수 있어 아주 즐거운 워크샵이었습니다.
- 모디: 잔잔하게 즐거웠던 시간이었던 것 같습니다 :)
- 디노: 다양한 사람들의 글을 읽음으로서 크루분들의 가치관을 엿볼 수 있었다는 게 즐거웠습니다.
- 제이: 다른 사람들이 생각하는 것들을 보면서 나도 저렇게 생각해야겠다, 따라가야겠다고 느꼈습니다.
- 테오: 저도 다른 크루분들의 생각을 들을 수 있다는 게 좋았고 또 이 기회를 통해서 조금 더 친해진 것 같아서 좋았습니다.

**Comment by cookienc** [comment]
## 13조 글쓰기 DAY2 회고
- 에단: 다들 칭찬을 많이 해주셔서 감사합니다. 덕분에 행복한 시간 보냈습니다. 꼭 작가가 된거 같네요.
- 바론: 다들 부둥부둥 잘해주셔서 몸둘바를 모르겠습니다... 기분이 정말 좋아지네요!  기분 좋아지는 방법에 추가하도록 하겠습니다. 다른 크루들의 글을 통해서 여러 가지 생각들을 다시 해볼 수 있었습니다."
- 조이: 칭찬을 들어서 좋습니다. 제 글의 부족한 점만 생각해봤는데, 좋은 부분에 대해서도 생각해볼 수 있었던 것 같아요. 저의 글에 대해 알아갈 수 있는 좋은 시간이었습니다!"
- 메리: 다른 사람들과 생각을 공유하는 것은 정말 즐거운 일입니다!
- 이오: 스스로 많이 부족한 글이라고 생각했는데 좋은 말씀을 많이 해주셔서 자신을 가질 수 있었습니다. 부족한 글이라도 다듬어서 완성해 보겠습니다.

**Comment by SproutMJ** [comment]
## 3조 글쓰기 DAY2 회고

* 엔델: 좋은 피드백을 받을 수 있었습니다.
* 마코: 즐거웠습니다.
* 주디: 제 글의 장단점을 알 수 있어서 좋았습니다.
* 엔초: 다른 분들 이야기 들으면서 재밌었고, 글을 어떻게 개선하면 좋을지 알게 되어 좋았다.

**Comment by BackFoxx** [comment]
# 17조 글쓰기 DAY2 회고
---
* 헤나 : 자신의 글에 대한 개선점도 찾고 크루들의 좋은 생각도 많이 들을 수 있었던 시간이다.
* 박스터 : 제 글에 대해 욕을 먹고싶었는데 아쉽습니다
* 두둠 : 다른 분들의 글을 볼 수 있어서 좋았습니다, 제 글도 보안할 수 있겠네요
* 여우 : 다양한 글을 읽을 수 있는 기회였어요
* 글렌 : 각자 다른 얘기들을 나눈게 좋았습니다저

**Comment by hafnium1923** [comment]
# 31조 글쓰기 day2 회고
- 코비 : 점심만 먹었으면 더 피드백을 알차게 해 드릴 수 있었을 텐데.. 아쉽습니다 힘이 없네요..
- 루루: 독자가 직접 앞에서 읽고 이야기 해주는 방식이 신선했습니다. 저도 객관적으로 제 글을 읽으려고 노력했던 것 같아요.
- 푸만능: 모든 글을 객관적으로 읽도록 노력하고 더 나은 글쓰기를 하기 위해 노력해야겠다. 
- 부엉이: 다양한 글을 읽을 수 있어 너무 좋았습니다!
- 룩소: 저의 글에서 생각하지 못한 여러 가지 피드백을 들을 수 있어서 좋았습니다. 또한 각자 크루들이 가지고 있는 여러 생각들을 글로 조금이나마 알아볼 수 있어 좋았습니닷.
- 멧돼지: 다양한 공감가는글들을 읽을수 있어서 좋았습니다

**Comment by waterricecake** [comment]
# 12조 글쓰기 day2 회고
- 가비: 제가 쓴 글에 대해 바로 소감을 들으니 좀 쑥스럽네요; ㅎㅎ
-  매튜: 내 글을 누군가가 진지하게 읽어주고 피드백 해주는 것이 즐거웠다.
- 오션: 생각하지 못했던 부분을 피드백을 통해 알아 차릴수 있어서 좋았다.
- 케로: 다른 크루들과 제가 쓴 주제에 대해 함께 이야기 해 볼 수 있어서 좋았습니다. 또 칭찬도 많이 해주셔서 감사하다 ^^*
- 달리: 누군가가 제 글을 읽는 다는것은 흔한 경험은 아닌데 이를 통해 글또한 말하기처럼 생각을 많이해야한다고 생각하게 됐다

**Comment by jeongwusi** [comment]
# 🌸 32조 글쓰기 day2 회고 🌼

- 산군: 공감가는 글을 많이 접해서 심신안정이 되었습니다.
- 파인: 좋은 글들을 읽을 수 있어서 좋았고 각자 피드백을 통해서 어떻게 글을 개선해 나갈 수 있을 지에 대해 영감을 얻어서 좋았습니다.
- 슬링키: 크루들의 다양한 글을 읽을 수 있어 정말 좋았습니다.
- 부나: 각자 어떤 상황이고, 무엇을 느끼고 있는지 이야기를 들을 수 있어서 좋았습니다~!
- 아인: 공감가는 글을 많이 읽어서 위로 받았고, 에너지도 많이 얻어가는 시간이었습니다.
- 고니: 각자의 진솔한 이야기를 듣게 되어서 너무 좋았습니다.

**Comment by hyemdooly** [comment]
## 30조 글쓰기 DAY2 회고
- 유스 : 다양한 글을 읽고 의견을 나눌 수 있어 즐거운 시간이었습니다! 양질의 피드백도 감사합니다
- 가브리엘 : 글을 다시 개선해보겠습니다
- 도담 : 다들 글을 잘 쓰시네요~! 서로서로의 정보를 공유하는 좋은 기회가 된 것 같아 행복합니다~!
- 둘리 : 정보공유글부터 회고까지 다양한 글을 읽을 수 있어서 재미있는 시간이었습니다.
- 에디 : 다양한 분들의 얘기들을 들어볼 수 있어서 좋았습니다. 저도 제 글을 개선하기 위해 노력해봐야겠습니다!

**Comment by shin-mallang** [comment]
## 16조 글쓰기 DAY2 회고

**Reply by shin-mallang** [reply]
말랑: 다른 크루들의 글을 보니 많은 교훈을 얻을 수 있어서 기분이 좋습니다.

**Reply by This2sho** [reply]
호이: 글을 읽어보지 못한 크루들의 글들을 읽어 볼 수 있어서 너무 좋았습니다.

**Reply by thdwoqor** [reply]
파워 : 다른 크루의 이야기를 들을수있어서 너무 좋았다.

**Reply by hong-sile** [reply]
홍실: 다른 크루들이 내 글에 대해 많은 피드백을 해줘서 글의 완성도를 높일 수 있을 것 같아서 좋았고, 다양한 생각을 가감없이 말할 수 있어서 좋았습니다.

**Reply by yujamint** [reply]
민트: 내가 글을 통해 전달하고자 했던 바가 잘 전달되었는지 확인할 수 있었다.
글 쓰는 것에 흥미를 못 느꼈는데, 다른 크루들의 글을 읽고 내 글을 피드백받을 수 있어서 흥미로웠다.
개발 외적인 부분도 다른 크루와 공유할 수 있다는 점에서 행복했다.

**Comment by nlom0218** [comment]
## 27조 글쓰기 회고

- 노아: 딱딱했던 글쓰기가 재밌는 말하기도와 함께 하니 지루한 틈이 없었습니다. 즐거운 시간이었습니다:)
- 패트릭: 다양한 크루들의 진심어린 글을 볼 수 있어 너무 좋았습니다. 또한 제 글을 읽고 솔직하게 말해주셔서 행복했습니다. 더 좋은 글이 나올 것 같고 오늘 하루의 시작이 참 좋습니다!!
- 타미: 다양한 종류의 글을 읽을 수 있어서 좋았습니다. 크루들의 이야기를 들을 수 있어서 재미있었습니다.
- 로피: 다양한 글들을 많이 읽을 수 있어서 좋았고 제 글에 대한 크루분들의 생각을 들을 수 있어서 좋았습니다!
- 아커: 크루들의 생각을 잠시나마 들을 수 있는 시간이어서 좋았습니다. 또한 제 글이 제 3자의 시각에선 어떠한지 피드백을 받을 수 있어 좋은 시간이었습니다!

**Comment by Combi153** [comment]
# 9조 글쓰기 회고
- 그레이 : 글쓰기 미션이 막막했는데, 다른 크루들과 이야기를 나눠보면서 글쓰기에 흥미를 더 가질 수 있던 계기가 되었습니다.
- 히이로 : 레벨1 때에도 워크숍으로 진행했으면 어땠을까 하는 아쉬움이 남을 정도로 좋았던 거 같아요! 바쁜 미션 속에서도 이런 생각들을 하고 계시는구나라고 느낄 수 있었어서 너무 좋았습니다. 다들 글 쓰시느라 고생 많으셨고 나중에 워크숍이 아니더라도 이렇게 생각을 공유할 수 있는 자리가 있었으면 좋겠습니다!
- 헙크 : 내가 보지 못했던 나의 부족한 점들을 알 수 있어서 좋았고, 크루들로부터 많이 얻을 수 있는 시간이었다.
- 필립 : 글을 적으며 고민을하다 그냥 넘어갔던 부분들이 있었는데, 팀원들이 좋은 의견을 제시해주고 하셔서 방향을 잡을 수 있어서 좋았습니다. 다른 크루들의 글을 읽으며도 많이 공감할 수 있어서 좋았습니다.
- 후추 : 크루들에게 글에 대한 의견을 들을 수 있어 좋았다. 또한 다른 크루들이 어떤 생각을 갖고 살고 있는 지에 대해 알 수 있어 좋았다. 모두 멋있는 사람들이다!

**Comment by cpot5620** [comment]
# 19조 글쓰기 day2 회고

- 스플릿 : 글을 많이 읽고 잘쓰는 것이 아니라서 그냥 이런 글이구나 재밌다 하면서 모두 다 읽었던 것 같다. 그냥 똑같은 주제와 같은 고민에 대해서 여러 말을 하게 된 것같아서 재밋었다.
- 로이스 : 다른 분들의 글을 깊이 있게 읽어볼 수 있어서 좋았습니다. 저 역시도 다양한 관점으로 글을 평가받을 수 있어서 좋았어요
- 제나 : 다른 사람들과 글을 공유하면서 부끄럽기도 했지만 개선방안도 알게 되고 다른 글들을 읽으면서 해당 크루의 생각을 더 알게 될 수 있어서 공감도 많이 할 수 있고 위로도 받고 리프레시도 된 거 같습니다.
- 제이미 : 글을 여러 사람과 공유하는 것은 언제 부끄럽고 수줍네요.. 그럼에도 좋은 이야기들과 좋은 피드백들을 받을 수 있어 좋은 시간이었습니다! 그리고 다른 분들의 이야기를 보면서 그들의 경험, 생각 등을 볼 수 있었어 재미있는 시간이었습니다 😄
- 깃짱 : 크루들 글도 재미있고, 각자 다 뜯어서 읽어보는게 굉장히 재미있었다. 많이 이야기해본 적이 없는 스플릿, 로이스랑, 제이미랑 같은 조가 되어서 이야기해볼 수 있어서 재미있었다. 
- 쥬니 :크루들의 생각, 상황 등을 들여다볼 수 있어서 좋았습니다. 글을 읽으며, 같이 웃고 슬퍼할 수 있는 기회였습니다.

**Comment by solo5star** [comment]
## 28조 글쓰기 day2 회고

윤생: 내 글에 대한 피드백을 텍스트가 아닌 면대면으로 받을수 있어 좋았다.
반달: 워크숍을 진행하면서 타인의 글쓰는 방법과 글을 쓸 때 중요한 가치관을 공유 할 수 있어서 좋았습니다.
가람: 아주 유익한 시간이었습니다! bb
쵸파: 너무 도움됐어요
솔로스타: 객관적으로 보는데에 큰 도움이 되네요~

**Comment by n0eyes** [comment]
# 24조 글쓰기 회고

- 노아이즈 : 내가 쓴 글을 소개한다는 것이 정말 부끄러웠지만 유익했습니다!
- 야미 : 다양한 사람들의 글을 읽을 수 있어서 좋았고, 많은 생각을 할 수 있게 되었습니다.
- 멘델 : 생각보다 다들 글을 유심히 읽어주신 것 같아서 좋았습니다
- 뽀또 : 크루들의 글을 읽고 다양한 정보들을 얻을 수 있어 좋았고 글을 어떻게 써야 좋을 지 알아갈 수 있어 너무 유익한 시간이였습니다~!
- 아이크 : 레벨 1에서와 달리 더 많은 사람에게 많은 피드백을 받을 수 있어서 글쓰는 것에 대한 어려움이 많이 줄었습니다!

**Comment by inyeong-kang** [comment]
## 22조 글쓰기 DAY2 회고
- 수달:  재밌는 이야기들을 들은 기분이라 즐거웠습니다. 제 글의 수정 방향을 잡을 수 있어 좋았습니다.
- 코난: 제 글에 대해서 자세하게 읽어주고 피드백 해주어서 좋았습니다. 글의 방향을 떠올리기 쉬워졌어요!
- 스캇: 다들 좋은말씀 해주셔서 많은 에너지를 얻고 가고, 다들 좋은 조언을 통해 글을 쉽게 수정할 수 있을 것 같아요! 
- 제로: 독자님들의 이야기가 흥미진진했어요
- 엽토: 다들 참 비슷하게 생활하고 있음을 느꼈습니다.

**Comment by beer-2000** [comment]
## 2조 글쓰기 DAY2 회고
- 체인저: 나의 글에 대한 개선점, 그리고 좋은 말씀들도 너무 좋았고, 다양한 사람들의 다양한 글, 그에 대한 다양한 의견을 들으며 많은 생각을 할 수 있어서 좋았습니다!
- 라온: 워크숍 하기 전에는 내 글이 별로라고 생각하여 글쓰기 싫었는데, 크루들의 칭찬을 듣다보니 또 만족스러워졌다. 다시 뭔가 힘을 내서 글 쓸 수 있을 것 같다.
- 블랙캣: 나의 글도 남에게 큰 도움이 될 수 있다는 것을 알 수 있어서, 앞으로도 글을 더 열심히 써야겠다는 생각을 했습니다.
- 누누: 한 명 한 명의 글에서 우테코에서 있을만한 다양한 에피소드를 볼 수 있어서 재밌었습니다. 우테코에서 사소한 부분을 다시 한번씩 되돌아보는 기회가 되었습니다. 이런 워크샵을 열어주신 우테코에 감사합니다.
- 토리: 늦게 참여해서 모든 크루들에 대한 리뷰를 하지는 못했지만, 짧은 시간에서도 다들 진심으로 코멘트를 남겨주시려는 모습이 느껴져 정말 좋았습니다. 주제가 나는 매일 생각하던 주제라 조금 지루하고 재미없다고 느껴졌는데, 다른 크루들에게 위로가 되었다고 해서 감동받고 글을 더 열심히 리팩토링 해봐야겠다는 생각이 들었다. 다들 너무 감사했습니다 🙇‍♀️ 워크숍 정말 좋은 것 같습니다.

**Comment by Leejin-Yang** [comment]
## 29조 글쓰기 DAY2 회고

- 황펭: 제 글에 대해서도 다른 시선들을 들어볼 수 있었습니다. 이야기하면서 받은 요약과 피드백이 큰 도움이 되었습니다. 어떻게 추가하고 개선할 지 감이 옵니다. 감사합니다!
- 에이든: 이번에는 글쓰기 주제가 다양해서 더욱 많은 얘기들을 할 수 있었습니다. 같은 주제에 대해 얘기하는 것도 좋지만 각자 이 시점에서 어떤 생각들을 가지고 있는지 알 수 있어 좋았습니다!
- 요술토끼: 많은 내용을 최대한 담아보고 싶었는데, 글을 읽는 독자들도 생각하면서 내용의 분량을 생각하고 글을 읽어야 하는 이유, 이 지식을 익혀야 하는 이유에 대한 필요성을 더 심어주면 좋을 것 같다는 생각이 들었습니다!
- 베리: 다른 분들의 글을 보고 피드백을 받으면서 어떻게 글을 써야 설득력 있는 글이 될지에 대해서 많은 것을 배웠습니다!
- 우코: 다들 글을 정말 잘 작성하시고, 개성이 넘치는 글을 읽는 게 좋았어요. 다들 피드백에도 진심이시고 덕분에 더 좋은 글을 만들 수 있을 것 같아요.

**Comment by apptie** [comment]
## 5조 글쓰기 DAY2 회고

- 하디: 글의 수정 방향성을 정할 수 있었던 것 같아 좋았다.
- 땡칠 : 꼭 진지하게 임하지는 않아도 원하는 바를얻을 수 있는 것 같다.
- 지토 : 어떤 식으로 글을 다듬어야 할지 알 수 있어서 좋았습니다.
- 져니 : 글을 열심히 수정해서 모두를 만족시킬만한 글을 작성해보겠습니다!
- 베베 : 다음에도 다 같이 글을 나누는 시간 가지면 좋겠어요!

**Comment by semnil5202** [comment]
### 33조 글쓰기 DAY2 회고
* 수아 : 1단계 글쓰기보다 더 자유로운 주제의 글이라서 좋았습니다.
* 링링 : 프런트 분들이랑 대화해볼 수 있는 시간이어서 좋았습니다.
* 세인 : 다들 반가웠습니다. 다양한 글에 대해 얘기해볼 수 있어서 좋았어요.
* 헤다 : 링링 써니 반가워요. 막상 하긴 싫지만 하면 좋은 글쓰기, 회고.
* 센트 : 여러 가지 주제의 글을 읽어볼 수 있어서 재밌었습니다.
* 써니 : 수아 세인 헤다 센트 너무 재밌었습니다.
